catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/geofeature/Feature.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.query.geofeature;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Base class for results returned from the FeatureService.
+ */
+public abstract class Feature {
```

Comment:
Was planning to ask someone about this at some point, possibly for next or future PR. I saw that there's lots of existing code dealing with shapes but didn't manage to find any structure I could return that can be all of three supported feature types.

I started down the road of trying to use only opengis interfaces (with the JTS implementations?) but it was looking verbose and I figured it could wait. I'm assuming that's the correct path, though, as we already use those elsewhere?